sdcm/mgmt.py

```diff
@@ -173,7 +173,8 @@ def is_status_in_list(self, list_status, check_task_progress=False):
         :return:
         """
         status = self.status
-        if check_task_progress and status != TaskStatus.NEW:  # check progress for all statuses except 'NEW'
+        if check_task_progress and status not in [TaskStatus.NEW,
+                                                  TaskStatus.STARTING]:  # check progress for all statuses except 'NEW' / 'STARTING'
             progress = self.progress
```

Comment:
where is a progress variable used?

Reply:
it's not used.

Reply:
@yarongilor so what is the point of this if? to printout/run `task progress`?

Reply:
the reasons are:
1) check that progress command works on varios task statuses ( that was how last manager bug found).
2) print the progress to log in cases needed for failures/performance analysis.

Reply:
It is a bit confusing and not understandable from the code. I would add a comment with a reason why this is done.

Reply:
@yarongilor please add the reasons that you mentioned as the comments to the code!!!

Reply:
done. comment added.